plugins/modules/ec2_vpc_dhcp_option.py

```diff
@@ -104,27 +110,76 @@
 '''
 
 RETURN = """
-new_options:
+changed:
+    description: Whether the dhcp options were changed
+    type: bool
+    returned: always
+dhcp_options:
     description: The DHCP options created, associated or found
-    returned: when appropriate
+    returned: when available
     type: dict
-    sample:
-      domain-name-servers:
-        - 10.0.0.1
-        - 10.0.1.1
-      netbois-name-servers:
-        - 10.0.0.1
-        - 10.0.1.1
-      netbios-node-type: 2
-      domain-name: "my.example.com"
+    contains:
+        dhcp_configurations:
+            description: The DHCP configuration for the option set
+            type: list
+            sample:
+              - '{"key": "ntp-servers", "values": [{"value": "10.0.0.2" , "value": "10.0.1.2"}]}'
+              - '{"key": "netbios-name-servers", "values": [{value": "10.0.0.1"}, {"value": "10.0.1.1" }]}'
+        dhcp_options_id:
+            description: The aws resource id of the primary DCHP options set created or found
+            type: str
+            sample: "dopt-0955331de6a20dd07"
+        owner_id:
+            description: The ID of the AWS account that owns the DHCP options set.
+            type: str
+            sample: 012345678912
+        tags:
+            description: The tags to be applied to a DHCP options set
+            type: list
+            sample:
+              - '{"Key": "CreatedBy", "Value": "ansible-test"}'
+              - '{"Key": "Collection", "Value": "amazon.aws"}'
 dhcp_options_id:
     description: The aws resource id of the primary DCHP options set created, found or removed
     type: str
     returned: when available
-changed:
-    description: Whether the dhcp options were changed
-    type: bool
-    returned: always
+new_options:
```

Comment:
Since ec2_vpc_dhcp_option_info returns `dhcp_options` and `dhcp_config` would it be better to rename this to dhcp_config, while returning new_options for backwards compatibility?

ie. return `dhcp_options`, `dhcp_config` and `new_options`, but only document dhcp_options and dhcp_config In theory we could then deprecate new_options with something like release 3.x, maybe with a parameter to choose to return it or not.

Reply:
Yeah that makes sense